index.html

```diff
@@ -43,8 +66,15 @@ <h1>Awesome Books</h1>
         </thead>
         <tbody id="book-list"></tbody>
       </table>
-    </div>
 
+      <div id="contact" class="d-none mt-5">
+        <div class="d-flex flex-column align-items-center">
+          <h2>contact us</h2>
+          <p>contact@gmail.com</p>
+          <p>&#169; Copyright by Hussein</p>
+        </div>
+      </div>
+    </div>
```

Comment:
- [ ] Nice Design however you forgot to add a footer in order to match wireframe so please make sure to add it 
<img width="299" alt="Screenshot 2023-05-03 at 19 03 56" src="https://user-images.githubusercontent.com/61790587/236005310-7957425d-b928-42c4-93ba-8e35770ba709.png">
<img width="962" alt="Screenshot 2023-05-03 at 19 05 44" src="https://user-images.githubusercontent.com/61790587/236005449-a81d7bf1-28ff-4542-ab32-932858d81ad4.png">
- Yours 
<img width="1436" alt="Screenshot 2023-05-03 at 19 05 58" src="https://user-images.githubusercontent.com/61790587/236005482-22748b81-5e0e-473d-9828-97dd5d8f8c99.png">
<img width="1435" alt="Screenshot 2023-05-03 at 19 06 06" src="https://user-images.githubusercontent.com/61790587/236005485-9b779c45-c936-4824-a0cd-42341ad310b1.png">